README.md

```diff
@@ -9,7 +9,9 @@ following aspects:
 * inhibiting alerts based on alert dependencies
 * aggregating alerts by labelset
 * handling notification repeats
-* sending alert notifications via external services (currently email or [PagerDuty](http://www.pagerduty.com/))
+* sending alert notifications via external services (currently email,
+[PagerDuty](http://www.pagerduty.com/) or
+[Pushover](https://www.pushover.net/))
```

Comment:
Oh! Sorry. Due to how it got displayed visually by GitHub in the side-by-side diff variant, I totally missed that. All good.